CMakeLists.txt

```diff
@@ -536,6 +536,8 @@ else()
     set(SWIFT_HOST_VARIANT_SDK_default "CYGWIN")
   elseif("${CMAKE_SYSTEM_NAME}" STREQUAL "Windows")
     set(SWIFT_HOST_VARIANT_SDK_default "WINDOWS")
+  elseif("${CMAKE_SYSTEM_NAME}" STREQUAL "Haiku")
+	set(SWIFT_HOST_VARIANT_SDK_default "HAIKU")
```

Comment:
Nit: indentation here is inconsistent with other lines; use two spaces instead of five.